packages/dds/merge-tree/api-report/merge-tree.api.md

```diff
@@ -630,21 +630,21 @@ export class LocalReferenceCollection {
     // @internal
     append(other: LocalReferenceCollection): void;
     // @internal (undocumented)
-    clear(): void;
-    // @internal (undocumented)
     createLocalRef(offset: number, refType: ReferenceType, properties: PropertySet | undefined, slidingPreference?: SlidingPreference, canSlideToEndpoint?: boolean): LocalReferencePosition;
     // @internal (undocumented)
     get empty(): boolean;
     // @internal
     has(lref: ReferencePosition): boolean;
-    // @internal (undocumented)
+    // @internal
     hierRefCount: number;
     // @internal (undocumented)
     isAfterTombstone(lref: LocalReferencePosition): boolean;
     // @internal (undocumented)
     removeLocalRef(lref: LocalReferencePosition): LocalReferencePosition | undefined;
     // @internal
     split(offset: number, splitSeg: ISegment): void;
+    // @internal
+    static validateRefCount?: (collection?: LocalReferenceCollection) => void;
```

Comment:
can you switch this to a module-level local + module-exported function to set it so it doesn't appear in the public API?

Ideally we'd also avoid adding overhead to the implementation in production (e.g. by doing some monkeypatching of methods in tests where we care), but it's not a big deal. I just don't want it in the API surface.

Reply:
You mean something like:
```ts
let validateRefCount;

export function setValidateRefCount(cb?: ...) {
  validateRefCount = cb;
}
```

Reply:
yup